source/api/commands/state.md

```diff
@@ -0,0 +1,148 @@
+---
+title: state
+
+---
+
+Get and set state _synchronously_.
+
+# Syntax
+
+```javascript
+.state()
+.state(key)
+.state(key, value)
+```
+
+## Usage
+
+### Getting
+
+```javascript
+var window = cy.state("window")
+```
+
+```javascript
+var location = cy.state("location")
+```
+
+You can also retrieve all of the existing state synchronously:
+
+```javascript
+var allState = cy.state()
+```
+
+### Setting
+
+```javascript
+cy.state("ids", [1, 2, 3])
+```
+
+The state with the key `ids` can subsequently be accessed synchronously.
```

Comment:
Hee hee... three mentions of the word _synchronously_ already 😆 I thought it best to hammer that point home.